spacy/ml/models/defaults/morphologizer_defaults.cfg

```diff
@@ -9,6 +9,6 @@ depth = 4
 embed_size = 7000
 window_size = 1
 maxout_pieces = 3
-subword_features = true
+subword_features = false
```

Comment:
Actually, now that I think about it, this is probably wrong in the definition of `HashCharEmbedCNN`: `subword_features` should not be a parameter of [hash_charembed_cnn](https://github.com/explosion/spaCy/blob/develop/spacy/ml/models/tok2vec.py#L73 ) but instead it should call `build_Tok2Vec_model` with `subword_features=False`. Then the morphologizer and sentrec defaults can remove this line for `subword_features`. Right now it's just an empty promise, because it's being set to `False` downstream anyway.

Reply:
(just saw the other PR - never mind about `sentrec`)